.prettierrc

```diff
@@ -11,7 +11,8 @@
         "add-field-many.graphql",
         "inline-many.graphql",
         "inline-many-list.graphql",
-        "test-grpc.graphql"
+        "test-grpc.graphql",
+        "tests/graphql/auth.graphql"
```

Comment:
Find innovative ways to not have to do this.

Reply:
I made an [issue](https://github.com/tailcallhq/tailcall/issues/945) for this